src/xpk/parser/storage.py

```diff
@@ -88,6 +89,81 @@ def add_storage_create_parser(
   )
 
 
+def add_storage_create_parser(
+    storage_subcommands_parser: argparse.ArgumentParser,
+) -> None:
+  storage_create_parser: argparse.ArgumentParser = (
+      storage_subcommands_parser.add_parser(
+          'create', help='create XPK Storage.'
+      )
+  )
+  storage_create_parser.set_defaults(func=storage_create)
+  req_args = storage_create_parser.add_argument_group(
+      'Required Arguments',
+      'Arguments required for storage create.',
+  )
+  add_shared_arguments(req_args)
+  req_args.add_argument(
+      '--access_mode',
+      type=str,
+      help='Access mode of created filestore instance',
+      default='ReadWriteMany',
+  )
+
+  req_args.add_argument(
+      'name',
+      type=str,
+      help='The name of storage',
+  )
+  req_args.add_argument(
+      '--vol', type=str, help='The name of the volume to create', required=True
+  )
```

Comment:
We probably should have some default value for this. Something like "default" or similar.